functests/1_performance/netqueues.go

```diff
@@ -309,9 +321,9 @@ func checkDeviceSupport(workernodes []corev1.Node, nodesDevices map[string]map[s
 		}
 	}
 	if noSupportedDevices {
-		Skip("Skipping Test: There are no supported Network Devices")
+		return false, err
 	}
-	return err
+	return true, err
```

Comment:
Why not instead of:
```
if noSupportedDevices {
		return false, err
	}
	return true, err
```
Just:
```return !noSupportedDevices, err``` ?

Reply:
I could use that, but this is clearer, the tests are already a bit too large in code